saifa/umbrel-app.yml

```diff
@@ -0,0 +1,36 @@
+manifestVersion: 1
+id: saifa
+category: finance,development tool
+name: Saifa - invoice lightning network manager
+version: "1.0.0"
+tagline: Simple, webhook for connect with invoice and invoice lightning network manager
+description: >-
+    Saifa is designed to enhance the interoperability and efficiency of applications interacting with the Lightning Network Daemon (LND).
+  Our project simplifies the integration process, allowing applications to seamlessly connect to LND through an intuitive webhook system. 
+  With Saifa, users can effortlessly create new invoices using a straightforward REST API, cancel invoices, and receive real-time updates on invoice status.
+  This system is crafted to streamline operations and facilitate a smoother transaction experience on the Lightning Network,
+  making it an essential tool for developers looking to optimize their applications for better performance and user satisfaction.
+releaseNotes: >-
+  - Base Project Launch: The foundational framework of our project has been successfully deployed, establishing the core functionalities and setting the stage for future enhancements.
+  - Webhook System Integration: We've introduced a robust webhook system designed to facilitate seamless updates on invoice statuses. This feature ensures that applications remain synchronized, providing timely notifications to users and other connected applications.
+  - User Interface for Invoice Management: A user-friendly interface has been implemented, allowing users to effortlessly monitor the status of their invoices. This intuitive UI simplifies the management process, ensuring a smooth and efficient user experience.
+  - Enhanced Invoice Operations: Users now have the ability to add and cancel invoices directly from the simple UI. This enhancement streamlines the invoice management process, making it more accessible and convenient for our users.
```

Comment:
For the initial release we can just leave these blank, since no one will get an app update notification.

You can change to:

`releaseNotes: ""`